docs/docs/gatsby-on-linux.md

```diff
@@ -2,12 +2,126 @@
 title: Gatsby on Linux
 ---
 
-> This is a TODO. Help our community expand it.
+# Linux
```

Comment:
We can omit this, as it's duplicating the page header. That way, there will be only one h1 per page (the frontmatter title generates one already).
```suggestion
```